src/sfctl/tests/app_test.py

```diff
@@ -11,10 +11,11 @@
 from mock import patch, MagicMock
 import sfctl.custom_app as sf_c
 
+
 class AppTests(unittest.TestCase):
     """App tests"""
 
-    def app_path_absolute_test(self):
+    def test_app_path_absolute(self):
```

Comment:
Is this because you changed the regex for matching test function names? Style wise the `*test` regex seems better than `test*`. Lexographically sorting is easier, and it follows that kind of module naming pattern where the `.test` module is a child of the original module.